docs/docs/user/migration-guide/v2-v3.md

```diff
@@ -236,3 +238,100 @@ This would make it more uniform with the existing `${component.<key>}` variables
 +     brokers: ${config.kafka_brokers}
 +     schemaRegistryUrl: ${config.schema_registry.url}
 ```
+
+## Summary
+
+!!! warning
```

Comment:
didn't we ignore these with 🤔 
```yaml
<!-- dprint-ignore-start -->
<!-- dprint-ignore-end -->
```

Reply:
Here it wasn't a problem apparently, but I did add the comments just in case